src/analyses/condVars.ml

```diff
@@ -1,4 +1,5 @@
 (** Must equality between variables and logical expressions. *)
+(* TODO: unused, what is this analysis? *)
```

Comment:
It's another untested analysis and the only use is in base analysis's `invariant`. But considering that we never activate it, this never affects the base analysis in our usage.
It's another must equality analysis, so I wonder how it's different from var_eq. Given that description, it sounds more specific and thus maybe cheaper than general var_eq?

Along with `MusteqDomain`'s unused code, this sounds like another candidate for eventual removal, but considering the mess of must equality analyses we currently have, it might be good to keep it for now. At some point a nice must equality analysis needs to be implemented, which is maintainable and covers all three of the existing ones, but that can be a future goal.